javascript/example/webgl_loader_draco.html

```diff
@@ -41,8 +41,8 @@
         </div>
         <pre id="fileDisplayArea"><pre>
     </div>
-    <script src="three.js"></script>
-    <script src="draco_decoder.js"></script>
+    <script src="https://cdn.rawgit.com/mrdoob/three.js/r84/build/three.min.js"></script>
+    <script src="../draco_decoder.js"></script>
```

Comment:
Yes, my changes to the instructions (attempt to) specify that the local server should be run from the root of the cloned folder, instead. Is this compatible with your own workflows?